doc/manual/configuration.rst

```diff
@@ -468,6 +468,33 @@ If a step depends on a certain variable then the result of the following step
 is already indirectly dependent on this variable. Thus it can be set during the
 following step anyway.
 
+
+checkoutAssert
+~~~~~~~~~~~~~~
+
+Type: List of checkoutAssertions
+
+Using checkoutAsserts you can make a build fail if a file content has
+been changed. This is especially useful to detect modifications in
+License files.
+
+The following properties are known:
+
+================= ========================================================================================
+Property           Description
+================= ========================================================================================
+file              | The file in the workspace to check.
+digestSHA1        | Digest of the file / part. Either pre calculate it using `sha1sum` command or take the
+                  | output of the first (failing) run. 
+start             | Optionally. Defaults to 1.
+end               | Optionally. Defaults to last line of file.
+================= ========================================================================================
+
+To assert only a part of the file hasn't changed use ``start`` and ``end``.
+Bob uses `sed` to grep the file content between ``start`` and ``end`` making it possible to use regular
```

Comment:
I'm not sure that this is a good idea. It ties Bob's behavior to the version of sed that is installed on the host. We might even run into locale and escaping issues. I would only support line numbers until a use case comes up that needs advanced features.

Reply:
I used `sed` before to get the content between `start` and `end`. I don't know a pure bash solution for this. Do you think it's ok to use `head` and `tail`? Or add this partial file hashing to the `bob-hash-engine`?

Reply:
I think using sed is ok. I would just restrict start and end to numbers. Exposing the more advanced features of sed might prove difficult for the future.